# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1534796102 25200
#      Mon Aug 20 13:15:02 2018 -0700
# Node ID 182e891789f56bbf5a3ce6a8a90d2fb02a021b4c
# Parent  2be7cf62b9ee2b58ee8379f2cdc91975c55222b1
fix libgui build failure with Qt 4.8

* main-window.cc (main_window::display_community_news): Fix libgui build
failure since cset ae2c9f1427f0 with Qt 4.8 using existing conditional.

diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -612,18 +612,22 @@ namespace octave
         m_community_news_window->resize (news_x, news_y);  // set size and center
         m_community_news_window->move ((win_x - m_community_news_window->width ())/2,
                                        (win_y - m_community_news_window->height ())/2);
       }
     else
       {
         // Window already exists, just update the browser contents
         QTextBrowser *browser
-          = m_community_news_window->findChild<QTextBrowser *>("OctaveNews",
-                                                  Qt::FindDirectChildrenOnly);
+
+          = m_community_news_window->findChild<QTextBrowser *>("OctaveNews"
+#if defined (QOBJECT_FINDCHILDREN_ACCEPTS_FINDCHILDOPTIONS)
+                                                               , Qt::FindDirectChildrenOnly
+#endif
+                                                              );
         if (browser)
           browser->setHtml (news);
       }
 
     if (! m_community_news_window->isVisible ())
       m_community_news_window->show ();
     else if (m_community_news_window->isMinimized ())
       m_community_news_window->showNormal ();
